packages/dev/core/src/Materials/Node/Blocks/loopBlock.ts

```diff
@@ -0,0 +1,143 @@
+import { NodeMaterialBlock } from "../nodeMaterialBlock";
+import { NodeMaterialBlockConnectionPointTypes } from "../Enums/nodeMaterialBlockConnectionPointTypes";
+import type { NodeMaterialBuildState } from "../nodeMaterialBuildState";
+import { NodeMaterialConnectionPointDirection, type NodeMaterialConnectionPoint } from "../nodeMaterialBlockConnectionPoint";

```

Comment:
`type` should not be used inside an `import` (that would be cool if we could tell VSC not to do that...). There are other occurrences in other files.